sequencing-server/src/lib/batchLoaders/activitySchemaBatchLoader.ts

```diff
@@ -14,13 +14,13 @@ export const activitySchemaBatchLoader: BatchLoader<
       activity_type(where: {
         _or: [
           ${keys
-            .map(
-              key => `{
+      .map(
```

Comment:
Was this format changed for a reason?

Reply:
I think it got hit by prettier, I didn't touch it manually.

Reply:
Is it using [this prettier file](https://github.com/NASA-AMMOS/aerie/blob/develop/sequencing-server/.prettierrc)?

Reply:
Ah, it was defaulting to using the language formatting. I fixed it in my IDE and it reverted the change.